torch/csrc/jit/passes/canonicalize_graph_fuser_ops.cpp

```diff
@@ -59,6 +59,7 @@ static void CanonicalizeOps(Block* block) {
           inputs.at(1) = new_other;
           Value* new_output =
               graph->insertNode(graph->create(it->kind(), inputs))->output();
+          new_output->node()->copyMetadata(*it);
```

Comment:
Why do we need to add copy metadata? Can we just copy type?

Reply:
to copy over `Scope`, `sourceRange` and `callstack` information as well. 

The copyMetadata for Node* does the above.
The copyMetadata for Value* will copy over `type` and `debugName`.

It appears we would need to call both depending on cases.